src/social-network-provider/facebook.com/UI/collectPosts.tsx

```diff
@@ -48,7 +48,10 @@ export function collectPostsFacebook(this: SocialNetworkUI) {
             const commentSelector = isMobileFacebook ? commentSelectorMobile : commentSelectorPC
 
             // ? inject comment text field
-            const commentBoxSelectorPC = root.clone().querySelectorAll<HTMLFormElement>('form form')
+            const commentBoxSelectorPC = root
+                .clone()
+                .querySelectorAll<HTMLFormElement>('[role="article"] [role="presentation"]')
```

Comment:
plz check selector in other language availability.

Reply:
Checked several languages, all work well.